### ISSUE TITLE
Menu System Navigation Flaw

### ISSUE THREAD

**Issue by AHBabington** [backstagetechnicalservices/website#84]
For example if you click on 'Events Diary' in the member's area you taken to the calender format page. This can be done without seeing the menu giving you access to 'My Diary', 'Event Sign-up', and 'Submit Event Report'. Could I suggest that a menu be added to all the sub pages so that one can change between them easily. For example, if you similarly click 'My Profile@ in the member's area 'My Events' and 'My Training' can be accessed from that page. After reasonable useage and looking around the website for it specifically I didn't actually realise 'Submit Event Report' existed because I never hover and just clicked and their is no option on the 'Events Diary' page.

**Comment by bnjns** [comment]
In order to click you have to hover ...

**Comment by potti15** [comment]
I'm not sure this is a major issue with the navigation itself, and I personally haven't noticed it to be a problem. If anything this could be resolved perhaps by an arrow on the menus that can expand, similar to Finder on OSX, as in http://puu.sh/nKl87/7b04956df5.png?

**Comment by AHBabington** [comment]
But not for significently long enough to realise information is placed on in a menu that appears on hovering. If you still don't understand I am happy to show you quickly afterwards as it is kind of hard to explain
----Original message----
From : notifications@github.com
Date : 17/03/2016 - 23:01 (GMTST)
To : website@noreply.github.com
Cc : ahb-backstage@mybtinternet.com
Subject : Re: [website] Menu System Navigation Flaw (#84)
In order to click you have to hover ...

**Comment by bnjns** [comment]
> If anything this could be resolved perhaps by an arrow on the menus that can expand, similar to Finder on OSX, as in http://puu.sh/nKl87/7b04956df5.png?

Good idea, certainly worth looking into. I'm going to have to re-style the navigation soon so I'll look into it then.

> If you still don't understand I am happy to show you quickly afterwards as it is kind of hard to explain

I don't need you to explain. What you'll quickly discover is I am not sympathetic to issues that are due to misuse rather than things not working how they should.

**Comment by AHBabington** [comment]
I agree an arrow would be an improvement but I still think the ability to navigate within the events section like the profile section without re-accessing the main menu would be beneficial.
----Original message----
From : notifications@github.com
Date : 17/03/2016 - 23:08 (GMTST)
To : website@noreply.github.com
Cc : ahb-backstage@mybtinternet.com
Subject : Re: [website] Menu System Navigation Flaw (#84)
If anything this could be resolved perhaps by an arrow on the menus that can expand, similar to Finder on OSX, as in http://puu.sh/nKl87/7b04956df5.png?
Good idea, certainly worth looking into. I'm going to have to re-style the navigation soon so I'll look into it then.
If you still don't understand I am happy to show you quickly afterwards as it is kind of hard to explain
I don't need you to explain. What you'll quickly discover is I am not sympathetic to issues that are due to misuse rather than things not working how they should.

**Comment by bnjns** [comment]
The real issue I have with this is you start to add a lot of potentially unnecessary detail that starts to clutter the page and the whole idea of this was to remove the clutter.

For the example of the events diary I was originally intended for people to be able to filter events on the diary (including ones they were doing) but I never got around to it. If you think this would be beneficial - please open another issue.
